ldms/src/core/ldms_xprt.c

```diff
@@ -223,6 +223,8 @@ ldms_t ldms_xprt_by_remote_sin(struct sockaddr_in *sin)
 	return 0;
 }
 
+extern void __put_share_lookup_ref(ldms_set_t rbd);
```

Comment:
By putting this code:
```
	ref_put(&rbd->set->ref, "share_lookup");
	ref_put(&rbd->ref, "share_lookup");
```
inside the __put_share_lookup_ref function, you lose the function name and line number where the reference was dropped.

Reply:
I will change the __put_shate_lookup_ref() function to be a #define. I wrapped the ref_put() calls into a function because both the share_lookup references of RBD and set must be put together.